setup.py

```diff
@@ -309,7 +309,7 @@ def write_version(filename: str = os.path.join(*[dirname(__file__), "airflow", "
     'pinotdb==0.1.1',
 ]
 postgres = [
-    'psycopg2-binary>=2.7.4',
```

Comment:
Why this ? Just wondering?

Reply:
Whoops this was not meant for this PR, removing it